GW2EIEvtcParser/EncounterLogic/Raids/W8/UraTheSteamshrieker.cs

```diff
@@ -74,7 +92,8 @@ internal override List<InstantCastFinder> GetInstantCastFinders()
     internal override void EIEvtcParse(ulong gw2Build, EvtcVersionEvent evtcVersion, FightData fightData, AgentData agentData, List<CombatItem> combatData, IReadOnlyDictionary<uint, ExtensionHandler> extensions)
     {
         var sulfuricEffectGUID = combatData
-            .Where(x => x.IsStateChange == ArcDPSEnums.StateChange.EffectIDToGUID && EffectGUIDs.UraSulfuricGeyserSpawn.Equals(x.SrcAgent, x.DstAgent))
+            .Where(x => x.IsStateChange == ArcDPSEnums.StateChange.EffectIDToGUID && EffectGUIDs.UraSulfuricGeyserSpawn
+            .Equals(x.SrcAgent, x.DstAgent))
```

Comment:
That is inside the "Where" if you want to line break it, add an extra tabulation, otherwise keep it as before.